graphql/resolvers.go

```diff
@@ -140,6 +142,25 @@ func (r *taskResolver) ReliesOn(ctx context.Context, at *restModel.APITask) ([]*
 	return dependencies, nil
 }
 
+func (r *taskResolver) Ami(ctx context.Context, at *restModel.APITask) (*string, error) {
+	d, err := distro.FindByID(*at.DistroId)
```

Comment:
There's a subtle difference, but this is actually not correct. This will give you the AMI that the distro is using currently, but the purpose of this and related tickets is to be able to display what AMI the task actually ran on. To do that you'll need to get the host that the task ran on and find the AMI in its distro field.

I would add a field to the REST model for this and do the logic there so that the API route can also use it